src/relay/op/tensor/transform.cc

```diff
@@ -1660,93 +1664,171 @@ Array<Integer> GetIntArray(Array<IndexExpr> arr) {
 
 // strided_slice
 TVM_REGISTER_NODE_TYPE(StridedSliceAttrs);
-bool StridedSliceRel(const Array<Type>& types, int num_inputs, const Attrs& attrs,
-                     const TypeReporter& reporter) {
-  CHECK_EQ(types.size(), 2);
-  const auto* data = types[0].as<TensorTypeNode>();
-  if (data == nullptr) return false;
 
+int64_t* ToVector(const runtime::NDArray& array) {
```

Comment:
Refactored in https://github.com/apache/incubator-tvm/pull/5459, and this can be removed after that PR is merged.